models/intel/smartlab-sequence-modelling-0001/README.md

```diff
@@ -0,0 +1,177 @@
+            <td rowspan=1>precision</td>
+            <td>0.38</td>
+            <td>0.94</td>
+            <td>0.77</td>
+            <td>0.65</td>
+            <td>0.6</td>
+            <td>0.85</td>
+            <td>0.56</td>
+            <td>0.68</td>
+            <td>0.74</td>
+            <td>0.88</td>
+            <td>0.72</td>
+            <td>0.78</td>
+            <td>0.69</td>
+            <td>0.7</td>
+            <td rowspan=2>0.77</td>
+        </tr>
+        <tr>
+            <td>recall</td>
+            <td>0.64</td>
+            <td>1</td>
+            <td>0.96</td>
+            <td>0.94</td>
+            <td>0.62</td>
+            <td>0.96</td>
+            <td>0.48</td>
+            <td>0.77</td>
+            <td>0.91</td>
+            <td>0.88</td>
+            <td>0.83</td>
+            <td>0.85</td>
+            <td>1</td>
+            <td>0.83</td>
+        </tr>
+    </tbody>
+</table>
+
+## Inputs
+The inputs to the network are feature vectors at each video frame, which should be the output of feature extraction network, such as i3d-rgb and resnet-50, and feature outputs of the previous frame.
+
+1. Input feature, name: `input`, shape: `1, 2048, 24`, format: `B, W, H`, where:
+
+   - `B` - batch size
+   - `W` - feature map width
+   - `H` - feature map height
+
+2. History feature 1, name: `fhis_in_0`, shape: `12, 64, 2048`, format: `C, H', W`,
+3. History feature 2, name: `fhis_in_1`, shape: `11, 64, 2048`, format: `C, H', W`,
+4. History feature 3, name: `fhis_in_2`, shape: `11, 64, 2048`, format: `C, H', W`,
+5. History feature 4, name: `fhis_in_3`, shape: `11, 64, 2048`, format: `C, H', W`, where:
+
+   - `C` - the channel number of feature vector
+   - `H'`- feature map height
+   - `W` - feature map width
+
+## Outputs
+
+The outputs also include two parts: predictions and four feature outputs. Predictions is the action classification and prediction results. Four Feature maps are the model layer features in past frames.
+1. Prediction, name: `output`, shape: `4, 1, 64, 24`, format: `C, B, H', W`,
+   - `C` - the channel number of feature vector
+   - `B` - batch size
+   - `H'`- feature map height
+   - `W` - feature map width
+After post-process with argmx() function, the prediction result can be used to decide the action type of the current frame.
+2. History feature 1, name: `fhis_out_0`, shape: `12, 64, 2048`, format: `C, H', W`,
+3. History feature 2, name: `fhis_out_1`, shape: `11, 64, 2048`, format: `C, H', W`,
+4. History feature 3, name: `fhis_out_2`, shape: `11, 64, 2048`, format: `C, H', W`,
+5. History feature 4, name: `fhis_out_3`, shape: `11, 64, 2048`, format: `C, H', W`, where:
+
+   - `C` - the channel number of feature vector
+   - `H'`- feature map height
+   - `W` - feature map width
+
+## Legal Information
+
+The original model is distributed under the following
+[license](https://github.com/fatchord/WaveRNN/blob/master/LICENSE.txt)
+
+```
+MIT License
```

Comment:
@yssemaev should smartlab models have the same license as all other intel models or they are distributed with their own?